shared/src/test/scala/uk/gov/homeoffice/drt/arrivals/ArrivalSpec.scala

```diff
@@ -0,0 +1,120 @@
+package uk.gov.homeoffice.drt.arrivals
+
+import org.specs2.mutable.Specification
+import uk.gov.homeoffice.drt.ports.SplitRatiosNs.SplitSources.{ApiSplitsWithHistoricalEGateAndFTPercentages, Historical}
+import uk.gov.homeoffice.drt.ports.{AclFeedSource, ApiFeedSource, ForecastFeedSource, LiveFeedSource}
+
+import scala.collection.SortedSet
+
+class ArrivalSpec extends Specification {
+  "Arrival bestPcpPaxEstimate" should {
+    val arrivalBase = ArrivalGenerator.arrival()
+    val liveFeedTotalPaxSource = TotalPaxSource(10, LiveFeedSource, None)
+    val portForecastFeedTotalPaxSource = TotalPaxSource(10, ForecastFeedSource, None)
+    val apiFeedWithSplitsTotalPaxSource = TotalPaxSource(10, ApiFeedSource, Some(ApiSplitsWithHistoricalEGateAndFTPercentages))
+    val apiFeedWithHistoricalSplitsTotalPaxSource = TotalPaxSource(10, ApiFeedSource, Some(Historical))
+    val apiFeedWithOutSplitsTotalPaxSource = TotalPaxSource(10, ApiFeedSource, None)
```

Comment:
I'm trying to think if we need this last field to be an option. Is there a real world use case corresponding to `None`?